src/tutorials/jbearing2.c

```diff
@@ -493,9 +493,10 @@ PetscErrorCode CallPermonAndCompareResults(Tao tao, void *ctx)
   PetscReal      rtol,atol,dtol;
   PetscInt       maxit;
   PetscReal      gatol, grtol, gttol;
+  PetscReal      rhs_norm;
+  PetscReal      tao_diff_tol = 1e2*PETSC_SQRT_MACHINE_EPSILON;
```

Comment:
Remove the assignment - the default is set later. Maybe it can be grouped with the PetscReal above.